specification/paths/Users-user_id.json

```diff
@@ -45,6 +46,7 @@
     "security": [
       {
         "OAuth2": [
+          "users.manage"
```

Comment:
@NickVries I'm thinking: we do not give users `users.manage` on their own user.

We could require the scope, but another way is to let the `/users/{user_id}` endpoint check the `user_id` from the current token and allow `getOne` when it matches, so we (and the brokers) do not have to give `users.manage` on everyone.

Reply:
@NickVries This endpoint was already silently protected by `users.manage`. It was not listed here because you can also query this endpoint without the scope, fetching your own user.

I think we should remove this line and still not mention it here, what do you think?